evalml/model_understanding/_partial_dependence_fast_mode_utils.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+
+from evalml.utils.woodwork_utils import infer_feature_types
+
+
+def _get_cloned_feature_pipelines(
+    features,
+    pipeline,
+    variable_has_features_passed_to_estimator,
+    X_training,
+    y_training,
+):
+    """Clones and fits pipelines for partial dependence fast mode.
+
+    Args:
+        pipeline (PipelineBase or subclass): Fitted pipeline that will be cloned.
+        features (string, tuple[int or string]): The target feature for which to create the partial dependence plot for.
+            If features is a string, it must be a valid column name in X.
+            If features is a tuple of int/strings, it must contain valid column integers/names in X.
+        X_training (pd.DataFrame, np.ndarray): The data that was used to train the original pipeline. Will
+            be used to train the cloned pipelines.
+        y_training (pd.Series, np.ndarray): The target data that was used to train the original pipeline. Will
```

Comment:
missing the docstring for `variable_has_features_passed_to_estimator`

Reply:
Thanks for catching--adding!